en/features/census_configuration.md

```diff
@@ -49,44 +52,38 @@ The information to be filled in is divided into three sections:
   ```
 
   Required fields for the request:
-
   - **Request method name**: Request method name accepted by the City Census WebService.
 
     Example:
     ![Request Data - Method name](../../img/remote_census/request-data-method-name-en.png)
-
   - **Request Structure**: Structure of the request received by the WebService of the Census of the City Council. The "static" values of this request should be reported. The "dynamic" values related to Document Type, Document Number, Date of Birth and Postal Code should be filled with null value.
 
     Example:
     ![Request Data - Structure](../../img/remote_census/request-data-structure-en.png)
     ![Request Data - Structure](../../img/remote_census/request-data-structure-info-en.png)
-
   - **Path for document type**: Path in the request structure that sends the Document Type.
 
-    *NOTE: DO NOT FILL IN if the WebService does not require the Document Type to verify a user.*
+     *NOTE: DO NOT FILL IN if the WebService does not require the Document Type to verify a user.*
```

Comment:
Why are we adding an extra space to this line? The "Example:" line below it doesn't have the extra space :thinking:.

Same thing in a couple of places below.